src/routes.ts

```diff
@@ -1 +1,3 @@
 export const SIGNIN = '/'
+export const DASHBOARD = '/dashboard'
+export const SIGNUP = '/criar-conta'
```

Comment:
Acredito que aqui e em sign in, deveriam se chamar `SIGN_UP` e `SIGN_IN`, assim está muito estranho de ler.